code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -107,6 +107,16 @@
 	new /obj/item/clothing/head/helmet/space/hardsuit/syndi(src)
 	return
 
+/obj/item/storage/box/syndie_kit/internals
+	name = "suspicious internals box"
+
+/obj/item/storage/box/syndie_kit/internals/New()
+	..()
+	new /obj/item/tank/emergency_oxygen/syndi(src)
+	new /obj/item/clothing/mask/gas/syndicate(src)
+	new /obj/item/reagent_containers/hypospray/autoinjector(src)
+	return
```

Comment:
Trailing return....

Reply:
Same code as the spacesuit/hardsuit boxes, should they be removed? Will update.